subsys/usb/usb_transfer.c

```diff
@@ -245,6 +248,9 @@ int usb_transfer(uint8_t ep, uint8_t *data, size_t dlen, unsigned int flags,
 	} else {
 		/* ready to read, clear NAK */
 		ret = usb_dc_ep_read_continue(ep);
```

Comment:
Please drop this commit. btw, line above is an other issue, mostly usb_dc_ep_read_continue from situation without and OUT event returns an error.

Reply:
dropped.